src/SeleniumLibrary/__init__.py

```diff
@@ -121,6 +121,10 @@ class SeleniumLibrary(DynamicCore):
     Notice that using the ``sizzle`` strategy or its alias ``jquery``
     requires that the system under test contains the jQuery library.
 
+    
+    This will Click Element containing ABC text. Notice that this approach can be error prone 
```

Comment:
I don't think this is worth of mention. We are trying to explain how the library is working and not trying to explain how xpath works.